README.md

```diff
@@ -101,6 +102,7 @@ To run tests, run the following command: Not Applicable
 ### Deployment
 
 You can deploy this project using GitHub Pages.
+Link: https://metaverse-nabeel.github.io/Portfolio-Project/
```

Comment:
- **[OPTIONAL]** Congratulations 👏🏽 your link is working as expected, please 🙏🏽 move the link to the correct section it should be placed on the LIVE DEMO section for professional documentation.
**HINT** `[Portfolio](https://metaverse-nabeel.github.io/Portfolio-Project/)`